libr/util/idpool.c

```diff
@@ -163,6 +162,67 @@ R_API void* r_id_storage_get(RIDStorage* storage, ut32 id) {
 	return storage->data[id];
 }
 
+R_API bool r_id_storage_get_lowest(RIDStorage *storage, ut32 *id) {
+	r_return_val_if_fail (storage, false);
+	ut32 i;
+	for (i = 0; i < storage->size && !storage->data[i]; i++);
+	*id = i;
+	return i < storage->size;
+}
+
+R_API bool r_id_storage_is_empty(RIDStorage *storage) {
```

Comment:
this is not used right now, is it?

Reply:
removed. also, the fd == 0 i was refering to the ridstorage code, i have fixed this remaining one in iofd .